hypervisor/dm/vpci/vpci.c

```diff
@@ -230,3 +230,281 @@ void vpci_cleanup(const struct acrn_vm *vm)
 		break;
 	}
 }
+
+static inline bool is_hostbridge(const struct pci_vdev *vdev)
```

Comment:
Add @pre for all those new APIs

Reply:
OK, will add, but looks misra c tool does not require this, so I do not add the @pre

Reply:
Added @pre to the new/modified APIs